File: Salon/src/main/java/com/example/salon/dto/BusinessResponse.java
package com.example.salon.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BusinessResponse {
    private Long id;
    private String name;
    private String slug;
    private Boolean active;
    private String bookingUrl;
    private LocalDateTime createdAt;
}

File: BlazorSalonApp/BlazorSalonApp/Dto/BusinessResponse.cs
namespace BlazorSalonApp.Dto;

public class BusinessResponse
{
    public long Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Slug { get; set; } = string.Empty;
    public bool Active { get; set; }
    public string BookingUrl { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
}